src/index.html

```diff
@@ -1,13 +1,63 @@
 <!doctype html>
-<html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+<html lang="en" class="home-page">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body class="home-page__content">
+
+    <div class="card" data-qa="card">
+
+      <img
+          src="./images/imac.jpeg"
+          alt="card__image"
+          class="card__image">
```

Comment:
Check indents
```suggestion
      <img
        src="./images/imac.jpeg"
        alt="card__image"
        class="card__image"
      >
```

Reply:
And will be better to specify more understandable alt text, imac, for example

Reply:
Yes, sorry I'm always overlook this part.